server/routerlicious/packages/services/src/mongodb.ts

```diff
@@ -488,21 +492,16 @@ export class MongoDbFactory implements core.IDbFactory {
 		);
 		// Need to cast to any before MongoClientOptions due to missing properties in d.ts
 		const options: MongoClientOptions = {
-			autoReconnect: true,
-			bufferMaxEntries: this.bufferMaxEntries ?? 50,
 			keepAlive: true,
 			keepAliveInitialDelay: 180000,
-			reconnectInterval: 1000,
-			reconnectTries: 100,
 			socketTimeoutMS: 120000,
-			useNewUrlParser: true,
```

Comment:
Are these options are all not available anymore? are there any instructions/pages to. show how to reconnect?